app/src/main/java/kr/ac/konkuk/gdsc/gdscsuyeon/ui/home/TodoAdapter.kt

```diff
@@ -0,0 +1,59 @@
+package kr.ac.konkuk.gdsc.gdscsuyeon.ui.home
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import kr.ac.konkuk.gdsc.gdscsuyeon.R
+import kr.ac.konkuk.gdsc.gdscsuyeon.data.Todo
+import kr.ac.konkuk.gdsc.gdscsuyeon.databinding.RowBinding
+
+class TodoAdapter(var todos: ArrayList<Todo>) : RecyclerView.Adapter<TodoAdapter.ViewHolder>(){
+
+    interface OnItemClickListener {
+        fun OnItemClick(data: Todo, position: Int)
+        fun OnTodoClick(data: Todo, position: Int)
```

Comment:
```suggestion
        fun onItemClick(data: Todo, position: Int)
        fun onTodoClick(data: Todo, position: Int)
```

함수 이름을 작성하실때는 camelCase로 작성하시는 것을 권장해드릴게요